curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c8ad0cd8f1e5201c4ef2e4.md

```diff
@@ -7,34 +7,24 @@ dashedName: step-56
 
 # --description--
 
-The `list` parameter is going to be the result of a query selector, which will return a `NodeList`. A `NodeList` is an array-like which contains the elements that match the query selector. 
+The `list` parameter is going to be the result of a query selector, which will return a `NodeList`. A `NodeList` is an array-like which contains the elements that match the query selector.
 
-You'll need to loop through these elements, so start by creating a `for` loop. Your iterator `i` should start at `0`, continue while it is less than the length of the list, and increment by `1` each iteration.
-
-# --hints--
-
-Your `getCaloriesFromInputs` function should have a `for` loop.
+You'll need to loop through these elements, and you can use a <dfn>for...of</dfn> loop to do this. The `for...of` loop, is used to iterate over elements in an iterable object like an array. The variable declared in the loop represents the current element being iterated over.
```

Comment:
NOTE:

Even though step 21 uses a regular `for` loop, I think we should still keep that as a review for campers for what they learned in the RPG game.

Then here we can introduce the `for...of` loop to show them another way to loop through an array